integrations/cdi/jedis-cdi/README.adoc

```diff
@@ -79,12 +79,87 @@ redis.clients.jedis.JedisPool.orders.port
 ----
 
 ...together with other similarly-named properties will result
-ultimately in a `JedisPool` object injectable under the name `orders`
-with a `port` property set to `6379` like so:
+ultimately in a `JedisPool` object injectable under the instance name
+of `orders` with a `port` property set to `6379` like so:
 
 [source,java]
 ----
 @Inject
 @Named("orders")
 private JedisPool pool;
 ----
+
+Property names that start with
+ `redis.clients.jedis.JedisPool.instanceName.` are parsed, and the
+ remaining portion of each name is treated as a Java Bean property of
+ https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/jedis/JedisPool.html[`JedisPool`],
+ or as a primitive value accepted by
+ https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/jedis/JedisPool.html#JedisPool-org.apache.commons.pool2.impl.GenericObjectPoolConfig-java.lang.String-int-int-java.lang.String-int-boolean-javax.net.ssl.SSLSocketFactory-javax.net.ssl.SSLParameters-javax.net.ssl.HostnameVerifier-[its
+ most specific constructor].  Because the `JedisPool` class inherits
+ from the
+ https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/util/Pool.html[`Pool`]
+ class, its writable Java Bean properties are available as well.
+
+Accordingly, the `JedisPool` properties that can be set are as
+follows, where `instanceName` should be replaced with the actual name
+used in application code:
+
+* `redis.clients.jedis.JedisPool.instanceName.clientName`
+* `redis.clients.jedis.JedisPool.instanceName.connectionTimeout`
+* `redis.clients.jedis.JedisPool.instanceName.database`
+* `redis.clients.jedis.JedisPool.instanceName.host`
+* `redis.clients.jedis.JedisPool.instanceName.password`
+* `redis.clients.jedis.JedisPool.instanceName.port`
+* `redis.clients.jedis.JedisPool.instanceName.socketTimeout`
+* `redis.clients.jedis.JedisPool.instanceName.ssl`
+
+Any documentation for these properties that exists may be found in the
+javadocs for the
+https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/jedis/JedisPool.html[`JedisPool`]
+and
+https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/util/Pool.html[`Pool`]
+classes.
+
```

Comment:
For more information about these properties, see the Javadocs for the https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/jedis/JedisPool.html[JedisPool]
and https://static.javadoc.io/redis.clients/jedis/2.9.0/redis/clients/util/Pool.html[Pool] classes.

Reply:
I'm curious to know why you wrote "any documentation ... that exists may be found".  Did you choose that language because we can't know sure whether any documentation actually exists for those properties at those third-party sites. :-)  I don't think we should be concerned about that possibility.